README.md

```diff
@@ -75,19 +75,27 @@ to ensure it's pointing to our server for the API server. Currently, any of the
 login functionality is not implemented. You can adapt the `.turbo/config.json`-file
 in the root of your mono repo.
 
+NOTE: You **don't** have to run `turbo login` step.
+
 ```json
 {
   "teamId": "team_blah",
   "apiUrl": "http://127.0.0.1:8080"
 }
 ```
 
+NOTE: `teamId` must start with "team_".
+
 After this you should be able to run `turbo` e.g. `turbo run build --force` to
 force the generating of new cache artefacts and upload it to our server.
 
 Alternatively, you can also use the arguments `--api="http://127.0.0.1:8080" --token="xxxxxxxxxxxxxxxxx"`
 
-The `teamId` in `.tubrbo/config.json` or the `--team`-argument for `turbo` CLI is
+Token can also be specified as enviroment variable `TURBO_TOKEN=xxxxx`, for example: `TURBO_TOKEN=xxxx turbo run build`
+
+NOTE: CLI argument `--team` is not supported, please only use `teamId` from `.turbo/config.json`.
+
+The `teamId` in `.turbo/config.json` is
```

Comment:
Lovely, thanks for adding these details :)

I think we need to add support for the `slug` query parameter looks like that's being used now. I will double check tomorrow